### DISCUSSION TITLE
Diverging vs 100% Stacked Bar Chart

### DISCUSSION THREAD

**Discussion by SrishtiPalani** [nmalkin/plot-likert#21] [Ideas]
Hello,
I wonder if instead of creating diverging stacked bars for showing percentages, we could also create 100% stacked bars? Diverging stacked bar charts are like stacked bar charts, except that they align the bars on a centre baseline instead of on the left and right.

(reference article: https://blog.datawrapper.de/divergingbars/)

 ![180306_divergent31](https://user-images.githubusercontent.com/11545354/130338160-e27fb32e-1b57-4b49-9e6d-f9e1e3879c61.gif)

**Comment by nmalkin** [comment]
Hmm, I find the argument in the blog post you linked fairly convincing, though it would be interesting to hear someone defend diverging stacked bars, since they do seem fairly popular. (For example, I think they're the default for the Likert R package, which this one is inspired by.)

To make a 100% stacked bar chart, it might actually be easier to skip this package altogether and use the matplotlib (or Pandas) `plot.barh` function directly. [This StackOverflow question](https://stackoverflow.com/q/23142358) (which is where this library started) actually has an example of how to do that, and I'm sure there are others.

I don't think I have the time to add this feature myself (at least at the moment), but I'd definitely accept a pull request incorporating 100% stacked bar charts as an alternate graphing option into this library.

**Comment by alaskamike** [comment]
I prefer diverging bars because the provide a sense of “balance” between positive and negative responses. The 100% might be useful to normalize scores when one question had significantly more responses than others. I don’t know when I would ever pull out neutral scores - “n/a” scores yes, but not neutral.

I think an option for the 100% plot would be good.

Mike
